artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/AddressInfo.java

```diff
@@ -130,6 +130,10 @@ public String toString() {
    }
 
    public boolean isInternal() {
-      return this.name.startsWith(ADVISORY_TOPIC);
+      return isInternal(this.name);
```

Comment:
This should be a boolean field internal, that is set by the protocol layer, we shouldn't have protocol specific bits in the core models.